### ISSUE TITLE
sender.history() returns only 100 last messages

### ISSUE THREAD

**Issue by vhdmsm** [luckydonald/pytg#60]
I want to get maximum number of messages what history command can return, but it only returns 100 last messages. For example this command:
```
sender.history('$05000000247d0a3cd1ee2ccf1cc3427a', 1000)
```

**Comment by luckydonald** [comment]
I believe this is limited by the CLI.
Did you try the command in the CLI?
(In text mode, as well as with the ```--json``` flag)

**Comment by vhdmsm** [comment]
Yes you're right, i tried this on cli and it didn't return 1000 last messages, is this telegram api limit or cli? anyway for increasing limit size?

**Comment by luckydonald** [comment]
Maybe looking at [the cli](https://github.com/vysheng/tg) can help you further, I currently have no time to dig there. Sorry.
